Google_Compute_Instance.yml

```diff
@@ -0,0 +1,888 @@
+      description: Creation timestamp
+      type: string
+    - contextPath: Gcp.Instance.InstanceName
+      description: The name of the resource, provided by the client when initially
+        creating the resource.
+      type: string
+    - contextPath: Gcp.Instance.Decription
+      description: An optional description of this resource.
+      type: string
+    - contextPath: Gcp.Instance.TagItems
+      description: 'Tags to apply to this instance. '
+      type: string
+    - contextPath: Gcp.Instance.MachineType
+      description: Full or partial URL of the machine type resource to use for this
+        instance
+      type: string
+    - contextPath: Gcp.Instance.Status
+      description: ' The status of the instance. '
+      type: string
+    - contextPath: Gcp.Instance.Zone
+      description: URL of the zone where the instance resides.
+      type: string
+    - contextPath: Gcp.Instance.SelfLink
+      description: Server-defined URL for this resource.
+      type: string
+    - contextPath: Gcp.Instance.DeletionProtection
+      description: Whether the resource should be protected against deletion.
+      type: boolean
+    - contextPath: Gcp.Instance.CanIpForward
+      description: Allows this instance to send and receive packets with non-matching
+        destination or source IPs.
+      type: boolean
+    - contextPath: Gcp.Instance.DiskType
+      description: Specifies the type of the disk
+      type: string
+    - contextPath: Gcp.Instance.NetworkInterfaces
+      description: 'An array of network configurations for this instance. '
+      type: string
+    description: Get the information of a certain instance
+  - name: gcompute-delete-instance
+    arguments:
+    - name: instanceName
+      required: true
+      description: Name of the instance resource to delete. For example, instance-1
+    - name: projectId
+      required: true
+      description: Project ID for this request. For example, foo-bar-123
+    - name: zone
+      required: true
+      description: The name of the zone for this request. For example, us-east1-b
+    outputs:
+    - contextPath: Gcp.Instance.ProjectId
+      description: The unique identifier for the resource.
+      type: string
+    - contextPath: Gcp.Instance.InstanceName
+      description: Name of the resource
+      type: string
+    - contextPath: Gcp.Instance.Status
+      description: 'The status of the operation, which can be one of the following:
+        PENDING, RUNNING, or DONE.'
+      type: string
+    - contextPath: Gcp.Instance.User
+      description: '  If errors are generated during processing of the operation,
+        this field will be populated.'
+      type: string
+    - contextPath: Gcp.Instance.Error
+      description: '  If errors are generated during processing of the operation,
+        this field will be populated.'
+      type: string
+    description: Delete a certain instance
+  dockerimage: fuxuyu/demisto-gcp:4
```

Comment:
Create docker image for Demisto, upload it to our DockerHub repo and uploaded the docker image files to our git content repo